cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/CpsFlowDefinition/help-script.html

```diff
@@ -23,3 +23,39 @@
     but if you specify multiple parameters they must all be named:
 </p>
 <pre>stage name: 'Build', concurrency: 1</pre>
+
+<h3>Environment Variables</h3>
+
+A set of environment variables are made available to all Jenkins Job types, including Workflow Jobs.
+The following is a general list of variables (by name) that are available to all Job types. See the notes
+below the list for Workflow specific details.
+
+<p>
+    <iframe src="${rootURL}/env-vars.html" width="100%"></iframe>
+</p>
+
+The following variables are only available inside <code>node</code>s (i.e. not available outside):
```

Comment:
In fact they are not currently available inside `node` either. As noted in JENKINS-27145:

> Setting these variables from `ExecutorStepExecution` would be welcome.